libs/ledger-live-common/src/ble/hooks/useBleDevicesScanning.ts

```diff
@@ -45,6 +45,16 @@ const DEFAULT_RESTART_SCANNING_TIMEOUT_MS = 4000;
  *
  * Note: if a communication is started with a device, the scanning should be stopped
  *
+ * The scanningBleError will depend on the bleTransportListen function. Some errors might not be thrown.
```

Comment:
what does it mean "some errors might not be thrown"? Is it "in some cases, no error will be thrown" as in "it will silently fail"?

Reply:
ok I read the rest of the comment and it's a bit clearer, but the wording here could be improved I think